src/auditlog/migrations/0008_logentry_additional_data_new.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.0.1 on 2019-12-20 16:01
```

Comment:
Is the library still compatible with Django 2.1 ?

Reply:
I'm not sure, why?

Reply:
Yes I made sure to test all the changes with darwin so it was compatible. I also went ahead and changed this library to use 2.1.3 as well